doc/manual/source/building.rst

```diff
@@ -0,0 +1,126 @@
+Building From Source
```

Comment:
Do we need a note saying that dnst is not yet published to crates.io so only building with `--git` is possible?